HW_6/task3/modul/chess.py

```diff
@@ -0,0 +1,33 @@
+# Задача 3
+
+# Добавьте в пакет, созданный на семинаре шахматный модуль. Внутри него напишите код,
+# решающий задачу о 8 ферзях. Известно, что на доске 8×8 можно расставить 8 ферзей так,
+# чтобы они не били друг друга. Вам дана расстановка 8 ферзей на доске, определите,
+# есть ли среди них пара бьющих друг друга. Программа получает на вход восемь пар чисел,
+# каждое число от 1 до 8 - координаты 8 ферзей. Если ферзи не бьют друг друга верните истину,
+# а если бьют - ложь.
+
+def check_queens(queens):
+    for i in range(len(queens)):
+        for j in range(i + 1, len(queens)):
+            x1, y1 = queens[i]
+            x2, y2 = queens[j]
```

Comment:
Можно сделать проще
```suggestion
    for i, (x1, y1) in enumerate(queens, 1):
        for x2, y2 in queens[i:]:
```
Все остальное изменять не нужно